server/src/views/includes/head.jade

```diff
@@ -19,6 +19,7 @@ head
   link(rel='stylesheet', href='https://npmcdn.com/angular-toastr/dist/angular-toastr.css')
   link(rel='stylesheet', href='https://fonts.googleapis.com/icon?family=Material+Icons')
   link(rel='stylesheet', href='https://use.fontawesome.com/releases/v5.1.0/css/all.css')
+  //- link(rel='stylesheet', href='https://ajax.googleapis.com/ajax/libs/angularjs/X.Y.Z/angular-animate.js')
```

Comment:
Is there a reason why we have a commented code here ?.

Reply:
It was a link I commented out while working on the card animation, will remove. Thanks for the feedback.